reference/pipeline/stages/index.md

```diff
@@ -169,6 +169,13 @@ Deploy a Kubernetes manifest YAML/JSON file.
 ### Find Artifacts From Resource
 Find artifacts from a Kubernetes resource.
 
+### Patch (Manifest)
+Update an already existing Kubernetes resource in place using the [Kubernetes patch operation](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/). Spinnaker can update the resource without knowing the entire resource (that is, you have to specify only the portion of the manifest you want to update). 
```

Comment:
nit: would you put line breaks here at the 80-100 character mark the way we do in the rest of the file?